modules/library/jdbc/src/main/java/org/geotools/jdbc/JDBCDataStore.java

```diff
@@ -712,7 +715,7 @@ public Integer getMapping(Class<?> clazz) {
      * <p>This method will map the classes of the attributes of <tt>featureType</tt> to sql types
      * and generate a 'CREATE TABLE' statement against the underlying database.
      *
-     * @see DataStore#createSchema(SimpleFeatureType)
+     * @see DataStore#createSchema(FeatureType)
```

Comment:
DataStore is designed to work only against SimpleFeatureType, if you want to use generic FeatureType then DataAccess is the API you want. Does not seem this change is correct.

Reply:
fixed